pkg/kubelet/kuberuntime/instrumented_services.go

```diff
@@ -180,7 +180,7 @@ func (in instrumentedRuntimeService) RunPodSandbox(config *runtimeapi.PodSandbox
 	const operation = "run_podsandbox"
 	startTime := time.Now()
 	defer recordOperation(operation, startTime)
-	defer metrics.RunPodSandboxLatencies.WithLabelValues(runtimeHandler).Observe(metrics.SinceInMicroseconds(startTime))
+	defer metrics.RunPodSandboxLatencies.WithLabelValues(runtimeHandler).Observe(metrics.SinceInSeconds(startTime))
```

Comment:
One question is, it seems `SinceInSeconds` may not be precise enough for latency of sandboxes? A kata vm can started in ~100ms easily.

Reply:
Here is the buckets in `prometheus.DefBuckets`:
https://github.com/kubernetes/kubernetes/blob/5b6a23f3af47ad746f50d89d69941a9c48de7a48/vendor/github.com/prometheus/client_golang/prometheus/histogram.go#L61

I did leave a comment in this PR mentioning that we may need to customize the buckets in the future. So far, I don't have enough data to do the customization.

Reply:
Thanks for explaining, it make sense now.